Dockerfile

```diff
@@ -103,3 +102,7 @@ RUN curl -sL http://get.sensiolabs.org/security-checker.phar -o security-checker
   && mv security-checker.phar /usr/local/bin/security-checker
 
 RUN ln -s /usr/bin/php /usr/local/bin/php
+
+WORKDIR /app
```

Comment:
@nilswloewen Do we need to have the WORKDIR & RUN lines here? Only one of the CI jobs should use npm & it explicitly changes to the node app root and runs npm install before testing.